src/App.tsx

```diff
@@ -15,48 +17,107 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortField {
+  SORT_FIELD_ALPHABET = 'alphabet',
+  SORT_FIELD_LENGTH = 'length',
+}
+
+interface SortParams {
+  sortField: string,
```

Comment:
please fix the typing for sortField everywhere
```suggestion
  sortField: SortField,
```